app/javascript/src/Navigation/components/header/DocumentationMenu.jsx

```diff
@@ -1,11 +1,21 @@
-import React from 'react'
+// @flow
+
+import * as React from 'react'
 import { DocumentationMenuItem } from 'Navigation/components/header'
+import { DocsProps } from 'Navigation/components/header/types'
```

Comment:
Same as comment above, also I think the proper name is `DocumentationMenuProps` or simply `Props` since it's private.